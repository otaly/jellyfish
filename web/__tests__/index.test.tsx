import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import Home from '../src/app/page';

describe('Home', () => {
  it('タイトルを表示する', () => {
    render(<Home />);

    const heading = screen.getByRole('heading', { level: 1 });

    expect(heading.textContent).toBe('Hello, World!');
  });

  it('スナップショットテスト', () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
