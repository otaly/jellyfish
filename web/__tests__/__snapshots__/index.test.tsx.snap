// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home スナップショットテスト 1`] = `
<div>
  <main
    class="flex min-h-dvh flex-col items-center p-16"
  >
    <h1>
      Hello, World!
    </h1>
  </main>
</div>
`;
